src/GUI/MenuItem.php

```diff
@@ -8,51 +8,18 @@
  */
 class MenuItem
 {
-    protected string $category;
-    protected string $label;
-    protected string $link;
-
     /**
      * Constructs a menu item in a given category.
      *
-     * @param string $category The category (dropdown section) for the menu item.
-     * @param string $label    The label to use for the menu item.
-     * @param string $link     The URL that the menu should point to.
-     */
-    public function __construct(string $category, string $label, string $link)
-    {
-        $this->category = $category;
-        $this->label    = $label;
-        $this->link     = $link;
-    }
-
-    /**
-     * Returns the category that the menu item should appear in
-     *
-     * @return string the category
-     */
-    public function getCategory() : string
-    {
-        return $this->category;
-    }
-
-    /**
-     * Returns the label that should be used for the menu item
-     *
-     * @return string the label
-     */
-    public function getLabel() : string
-    {
-        return $this->label;
-    }
-
-    /**
-     * Returns the URL to use for the menu item
-     *
-     * @return string the URL.
+     * @param MenuCategory $category The category (dropdown section) for the
+     *                               menu item.
+     * @param string $label          The label to use for the menu item.
+     * @param string $link           The URL that the menu should point to.
      */
-    public function getLink() : string
-    {
-        return $this->link;
+    public function __construct(
```

Comment:
Forgot to mention in the description--I also simplified/cleaned up the existing LorisMenu class with more modern PHP features (constructor property promotion and read-only properties) while I was there.